app/helpers/ndr_ui/bootstrap_helper.rb

```diff
@@ -243,6 +243,34 @@ def bootstrap_form_for(record_or_name_or_array, *args, &proc)
       end
     end
 
+    # Identical signature to form_with, but uses NdrUi::BootstrapBuilder.
+    # See ActionView::Helpers::FormHelper for details
+    def bootstrap_form_with(**options, &block)
+      options[:html] ||= {}
+      options[:builder] = NdrUi::BootstrapBuilder
+      horizontal = options.delete(:horizontal)
+
+      # :horizontal
+      if horizontal
+        # set the form html class for horizontal bootstrap forms
+        options[:html][:class] ||= ''
+        classes = (options[:html][:class].split(' ') << 'form-horizontal').uniq.join(' ')
+        options[:html][:class] = classes
+      end
+
+      # We switch autocomplete off by default
+      raise 'autocomplete should be defined an html option' if options[:autocomplete]
+      options[:html][:autocomplete] ||= 'off'
+
+      form_with(**options) do |form|
+        # Put the form builder into horizontal mode (if necessary)
+        form.horizontal_mode = horizontal if horizontal
+
+        # yield to the provided form block
+        block.call(form)
```

Comment:
RedundantBlockCall warning v.s. UnusedMethodArgument warning??

By the way, is there any way I can check what hound bot would check locally before the git push? rubocop??

Reply:
I tend to prefer using explicit `&block` rather than it being implicit and using `yield` / `block_given?` - it makes it clear in the signature how the method functions.